tox.ini

```diff
@@ -1,7 +1,22 @@
 [tox]
-envlist = py33, py34, py35
+package=gobble
+skip_missing_interpreters=true
```

Comment:
Not much point in skipping missing interpreters here, as there's only one interpreter (i.e. if you don't have py36, you can't run the tests)